### PR TITLE
branchES!

### PR DESCRIPTION
**Pull request by amandacarson**
deleted things that stopped being true

### CODE REVIEW

File: facts.csv [hunk 1]
```diff
@@ -1,14 +1,4 @@
 id,category,fact
-1,law,"89% of males will be attacked by a mountain lion on their 25th birthday"
-2,law,"45% of children turn into bears by the age of 14"
-3,law,"0.002% of people ever fall in love"
-4,law,"20% of people name Sam own their own small island near New Zealand"
-5,law,"Only 13% of people know how to swim, or even what an ocean is"
-6,law,"Less than 1% of people know more than 3 countries in Africa by name and location"
-7,law,"Only 4% of people with the initial KMG will live more than a year after they graduate"
-8,law,"Only 7 people have ever been able to ride a bike without someone holding on and running after it"
```
Comment (amandacarson) [inline facts.csv:9]: this is still true

**Comment by ryanpbrewster** [comment]
I think we should drop the `id` column entirely. It's an autoincrement field, it doesn't need to be supplied.

**Comment by zekeage** [comment]
Leave it for now, my magic table reading routine uses it

**Comment by ryanpbrewster** [comment]
@zekeage I don't think that's actually true. If you're just loading in to MySQL from CSV, it can figure out the `id` on its own.

**Pull request merged**